1-js/02-first-steps/06-type-conversions/article.md

```diff
@@ -30,7 +30,7 @@ alert(typeof value); // string
 
 String conversion is mostly obvious. A `false` becomes `"false"`, `null` becomes `"null"`, etc.
 
-## ToNumber
+## Numberic Conversion
```

Comment:
Numberic -> Numeric

Reply:
Thanks for spotting out my typo. Fixed now